src/commands/build.rs

```diff
@@ -388,11 +409,12 @@ pub async fn build(
         .repository(git_repo)
         .build()
         .setup()
+        .instrument(build_span.clone())
         .await?;
 
-    info!("Running orchestrator...");
+    info!(parent: &build_span, "Running orchestrator...");
     let mut artifacts = vec![];
-    let errors = orch.run(&mut artifacts).await?;
+    let errors = orch.run(&mut artifacts).instrument(build_span).await?;
     let out = std::io::stdout();
     let mut outlock = out.lock();
 
```

Comment:
Question: Would it make sense to `enter()` `command_span` for the rest to get an accurate end time (shouldn't make much of a difference though...).

Reply:
No, the span is dropped anyways at the end of the scope.

Reply:
Ok, should probably be fine then - my assumption was that the end time of `command_span` would be after this `let errors = orch.run(&mut artifacts).instrument(build_span).await?` at line 418 when not using `enter()` but that doesn't make much sense when thinking about it as it would create other issues.